packages/odyssey-design-tokens/src/color/text.json

```diff
@@ -5,6 +5,14 @@
         "comment": "Base text color used for most typography",
         "value": "{color.palette.neutral.900.value}"
       },
+      "sub": {
```

Comment:
We now have three `sub` entries. I'll fix in this PR.

Reply:
thanks - my thought was it was more appropriate to reference the `neutral` functional color rather than reach all the way back to the palette color.